src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,22 @@
+.catalog {
+  max-width: ($card-width * 4) + ($card-right-margin * 3);
+  min-width: $card-width * 4;
+  justify-content: space-between;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin: auto;
```

Comment:
Move margin to `page__catalog`, because we couldn't  set margins for blocks, only for elements
![Screenshot 2022-07-21 at 20 39 14](https://user-images.githubusercontent.com/17098736/180278235-59604410-9d02-4f01-a9d0-95aa82fe3fb0.png)
.

Reply:
Thank you! Changes implemented!